crates/dbs-miniball/src/vm-vcpu/src/vm.rs

```diff
@@ -15,7 +15,8 @@ use kvm_bindings::{
 };
 
 use kvm_ioctls::{Kvm, VmFd};
-use vm_device::device_manager::IoManager;
+// use vm_device::device_manager::IoManager;
```

Comment:
Why keep the `// use vm_device::device_manager::IoManager;` here?

Reply:
sorry, I forgot to delete it.